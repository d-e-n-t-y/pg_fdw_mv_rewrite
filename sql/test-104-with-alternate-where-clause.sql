explain (VERBOSE, COSTS OFF) select key, COUNT (value) from test_lb where hidden = 'hidden3' group by key;

-- FIXME: although the balooning of WHERE cluases seems harmless, we should find out why it is happening.
select key, COUNT (value) from test_lb where hidden = 'hidden3' group by key;
