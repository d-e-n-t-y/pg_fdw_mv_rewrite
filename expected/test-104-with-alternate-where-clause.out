explain (VERBOSE, COSTS OFF) select key, COUNT (value) from test_lb where hidden = 'hidden3' group by key;
                                                                                                 QUERY PLAN                                                                                                 
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 Foreign Scan
   Output: key, (count(value))
   Relations: Aggregate on (public.test_lb)
   Remote SQL: SELECT key, count(value) FROM public.test WHERE ((hidden = 'hidden3'::text)) AND ((hidden = 'hidden3'::text)) AND ((hidden = 'hidden3'::text)) AND ((hidden = 'hidden3'::text)) GROUP BY key
(4 rows)

-- FIXME: although the balooning of WHERE cluases seems harmless, we should find out why it is happening.
select key, COUNT (value) from test_lb where hidden = 'hidden3' group by key;
 key | count 
-----+-------
(0 rows)

